transmart-core-db/grails-app/domain/org/transmartproject/db/dataquery/highdim/rnaseq/transcript/DeRnaseqTranscriptData.groovy

```diff
@@ -0,0 +1,38 @@
+package org.transmartproject.db.dataquery.highdim.rnaseq.transcript
+
+import groovy.transform.EqualsAndHashCode
+import org.transmartproject.core.dataquery.highdim.rnaseq.RnaSeqValues
+import org.transmartproject.db.dataquery.highdim.DeSubjectSampleMapping
+
+
+@EqualsAndHashCode(includes = ['assay', 'transcript'])
+class DeRnaseqTranscriptData implements RnaSeqValues, Serializable {
+
+    Integer readcount
+    Double normalizedReadcount
+    Double logNormalizedReadcount
+    Double zscore
+
+    static belongsTo = [
+            assay     : DeSubjectSampleMapping,
+            transcript: DeRnaseqTranscriptAnnot
+    ]
+
+
+    static mapping = {
+        table schema: 'deapp'
+
+        id composite: ['assay', 'transcript']
```

Comment:
I know that this works, because the column naming in `transmart-data` is consistent with the default naming strategy of Hibernate. But to be consistent with the rest of the codebase, I would use explicit mapping of table name and column names.
E.g., adding `table schema: 'deapp', name: 'de_rna_seq_transcript_data'` and `assay column: 'assay_id'` and `transcript column: 'transcript_id'`.